src/abi/global.ts

```diff
@@ -174,32 +175,40 @@ export const GlobalFunctions: Map<string, AbiFunction> = new Map([
                     return `${ctx.used("__tact_nop")}()`;
                 }
                 const arg = args[0];
+
+                const filePath = ref.file
+                    ? path.basename(ref.file!)
```

Comment:
I think we should print the filename relative to the current working directory. This would make it easier to access this file from the terminal

Reply:
<img width="576" alt="image" src="https://github.com/tact-lang/tact/assets/2316541/be995734-0ecf-41ea-afc3-037eb8a02ed8">

Reply:
this is the difference between Tact's `dump` and the output from Jest